api.go

```diff
@@ -474,45 +474,45 @@ func DeleteContainer(podID, containerID string) (*Container, error) {
 
 // StartContainer is the virtcontainers container starting entry point.
 // StartContainer starts an already created container.
-func StartContainer(podID, containerID string) (*Container, error) {
+func StartContainer(podID, containerID string) (*Pod, *Container, error) {
```

Comment:
@sboeuf so then we should add a `container.URL()` method to fetch the URL a container is reachable through (i.e. its Pod URL). `Container` points back to its Pod, so we should use that to avoid complicating our API.